uw-frame-components/portal/widgets/partials/compact-widget-card.html

```diff
@@ -0,0 +1,17 @@
+<md-card class="list-content" id="widget-id-{{::widget.nodeId}}">
+  <md-button class="widget-action widget-info md-icon-button" aria-label="see a brief description of this app">
```

Comment:
Good catch.  I just copied and pasted a partial from before.  It's always good to be better.  Updated.